packages/animation-curve-block/src/components/Card.tsx

```diff
@@ -171,6 +174,7 @@ export const Card = forwardRef<HTMLDivElement, CardProps>(
                             setDescription={onDescriptionChange}
                             updateValueOnChange={false}
                             isEditing={isEditing}
+                            textShown={textShown}
```

Comment:
`{textShown && <CardText ... />}` instead of passing it as props.

Reply:
You are right thats much better, thanks